en/docs/install-and-setup/setup/deployment/kubernetes-deployment.md

```diff
@@ -6,8 +6,6 @@ You can set up a Kubernetes (K8s) cluster for the Micro Integrator using one of
 
 - [Using Helm resources](#using-helm-resources)
 
-- [Using pure Kubernetes resources](#using-pure-kubernetes-resources)
```

Comment:
Please rephrase "You can set up a Kubernetes (K8s) cluster for the Micro Integrator using one of the following deployment options."